src/EditorFeatures/Core/Implementation/InlineRename/IEditorInlineRenameService.cs

```diff
@@ -205,6 +226,15 @@ internal interface IInlineRenameInfo
         bool TryOnAfterGlobalSymbolRenamed(Workspace workspace, IEnumerable<DocumentId> changedDocumentIDs, string replacementText);
     }
 
+    internal interface IIInlineRenameInfoWithFileRename : IInlineRenameInfo
```

Comment:
this LGTM.  note: `III`

Reply:
It's a really meta COM naming style, `II` `InlineRenameInfoWithFileRename` ~= `IInlineRenameinfo2` 

Thanks for pointing it out :)